### PR TITLE
Added bash script to get imu data with screen

### PR DESCRIPTION
**Pull request by luisdavidgarcia**
Split the parsing from the bash script since it was simpler in python to parse. Also kept it pretty minimal making the only two accepted arguments:

1. Serial Port
2. Baud Rate

### CODE REVIEW

File: scripts/capture_and_parse_imu_serial_data.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Set the default serial port, baud rate, and capture duration (in seconds)
+DEFAULT_SERIAL_PORT="/dev/cu.usbmodem2101"
```
Comment (kennedyengineering) [inline scripts/capture_and_parse_imu_serial_data.sh]: Maybe make OS agnostic? For linux this is /dev/ttyACM0. Could perform check in script.

Comment (kennedyengineering) [reply scripts/capture_and_parse_imu_serial_data.sh]: Not a huge worry

Comment (luisdavidgarcia) [reply scripts/capture_and_parse_imu_serial_data.sh]: I gotchu that way we can just use the defaults

**Comment by kennedyengineering** [comment]
Made a few organizational changes in the last two commits, file structure wise. Didn't actually change any code. Also, updated with master branch so it is up to date.

File: src/Arduino/main/main.ino
Comment (kennedyengineering) [inline src/Arduino/main/main.ino:1]: Ignore this, artifact from updating to master

File: src/Scripts/parse_screen_imu_log.py
Comment (kennedyengineering) [inline src/Scripts/parse_screen_imu_log.py:1]: Think about putting this in the src/Python? Should we remove logger.py?

File: logs/.gitignore
Comment (kennedyengineering) [inline logs/.gitignore:1]: Created log/ which is tracked in repo

**Comment by kennedyengineering** [comment]
Maybe make the parse script a separate MR. Usually one feature per pull request.

File: src/Scripts/parse_screen_imu_log.py
Comment (luisdavidgarcia) [reply src/Scripts/parse_screen_imu_log.py:1]: We could remove logger.py since we mainly go the bash and parser script for it

**Pull request closed without merging**